manager/integration/tests/test_migration.py

```diff
@@ -15,38 +14,29 @@ def test_migration_confirm(clients, volume_name):  # NOQA
 
 
 def migration_confirm_test(clients, volume_name, base_image=""):  # NOQA
-    client = get_random_client(clients)
-    hosts = clients.keys()
-    host1 = hosts[0]
-    host2 = hosts[1]
-
-    volume = client.create_volume(name=volume_name, size=SIZE,
-                                  numberOfReplicas=REPLICA_COUNT,
-                                  baseImage=base_image)
-    volume = common.wait_for_volume_detached(client, volume_name)
+    client, volume, data = setup_migration_test(clients, volume_name,
+                                                base_image)
+    host1, host2 = get_hosts_for_migration_test(clients)
 
     volume.attach(hostId=host1)
     volume = common.wait_for_volume_healthy(client, volume_name)
 
-    volume = volume.migrationStart(nodeId=host2)
-    attached_nodes = get_volume_attached_nodes(volume)
-    assert host1 in attached_nodes
-    assert volume["migrationNodeID"] == host2
+    volume.attach(hostId=host2)
     with pytest.raises(Exception) as e:
-        volume.migrationConfirm()
+        volume.detach(hostId=host1)
     assert "migration is not ready" in str(e.value)
```

Comment:
Is this for checking a transient state? If YES, we need to remove this. Same below